### ISSUE TITLE
When file is uploaded to a work, show the Representative Media and Thumbnail selectors

### ISSUE THREAD

**Issue by hannahfrost** [samvera-labs/hyrax#1148]
After at least one file has been successfully uploaded to a new work, the Representative Media and Thumbnail selections should appear so that the user can see what is available for selection.

![a2b0498c-4a0d-11e7-80f3-a387c221198d](https://user-images.githubusercontent.com/1801076/26843245-6b864456-4aa4-11e7-8d23-b43c68dcc7e9.png)

### Related work

Decomposed from https://github.com/samvera-labs/hyku/issues/1188
Depends on #1147

**Comment by atz** [comment]
Non-visual (filename-based) selection is sub-optimal.  When you have, say, 300 timestamp-named images, a dropdown is a rather unusable interface for selection.  Ideally, some indication of representative/thumbnail status should be IN the row.  "Set Representative" and "Set Thumbnail" are perfectly intelligible "Actions" one might take on an image (row).

In particular, paired with the separate ticket to "warn when deleting thumb/representative image", it makes sense to put status info close to the Delete button and close to a visual presentation of the image itself.

**Comment by ggeisler** [comment]
@atz @mjgiarlo Shouldn't this suggestion be a new ticket (e.g., "Improve UX for selection of representative thumbnail" or something)?

I think this ticket and #1147 were created just to get the thumbnail selection in a better place. Creating a more optimal UX for this feature sounds good, but requires more design (and probably development) time than completing the intended goal of this ticket. Do we want to block progress on the goal of this ticket by putting it back in the design queue?

(There are many, many features in the app that could use a more optimal UX but blocking work on useful incremental improvements to wait for additional designs doesn't seem like the best approach to me, given I seem to be the only one contributing designs and I'm working on other projects.)

**Comment by atz** [comment]
I'm tired of the app looking and behaving poorly.  We clearly are not going to get through all the tech debt we have already created for ourselves, so deferral might as well be disavowal.  Adding more debt in the name of expediency is exactly how we got into this situation.  I don't see why we can't build what we want instead of a provisional unusable approximation.
